frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -1310,9 +1315,23 @@ public JSONArray getComponentData(List<Component> componentList, PaginationParam
             jsonObject.put("attsSize", String.valueOf(comp.getAttachmentsSize()));
 
             JSONArray vendorArray = createJSONArray();
+            Set<String> vendorNames = new HashSet<>();
+            if (comp.isSetDefaultVendorId()) {
+                Vendor defaultVendor = null;
+                try {
+                    defaultVendor = vendorClient.getByID(comp.getDefaultVendorId());
+                } catch (TException e) {
+                    log.error("Could not get vendor for id [" + comp.getDefaultVendorId() + "] in component with id ["
+                            + comp.getId() + "] because of: ", e);
+                }
+                if (defaultVendor != null) {
+                    vendorNames.add(defaultVendor.getShortname());
+                }
+            }
             if (comp.isSetVendorNames()) {
-                comp.getVendorNames().stream().sorted().forEach(vendorArray::put);
+                vendorNames.addAll(comp.getVendorNames());
             }
+            vendorNames.stream().sorted().forEach(vendorArray::put);
```

Comment:
I would do the "jsonObject.put("vndrs", ...) after this line to make it easier to read.

Reply:
[x] done